disable_deactivated_accounts.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python
+import sys
+import optparse
+import requests
+import ldap
+import json
+
+from auth0_api_settings import AUTH0_API_KEY, AUTH0_API_URL, AUTH0_CONNECTION, LDAP_HOST,\
```

Comment:
recommend using a configuration file instead of a python file (ex: https://github.com/gdestuynder/rra2json/blob/master/rra2json.py#L219)